docs/tutorial.md

```diff
@@ -107,7 +107,7 @@ To configure Ruff, let's create a `pyproject.toml` file in our project's root di
 
 ```toml
 [tool.ruff]
-# Set the maximum line length to 79 characters.
+# Set the maximum line length to 79.
```

Comment:
I'm undecided if we should remove `characters` from here. I don't think mentioning it is necessary, most will assume characters anyway.

Reply:
I'm in favor of removing it